### ISSUE TITLE
Unable to load images

### ISSUE THREAD

**Issue by PrinceDhaliwal** [onlineteamPEC/Pecfest_2016_website#1]
I am using __WAMP__ on windows. But I am unable to load images. I read `index.php` and saw that images were being fetched from `img/homeSection1/row1` directory which is not in repository. May be you forgot to upload the folder.

**Comment by harshita2405** [comment]
Yes , i forgot that . I've uploaded the folder now , you can check.
